### PR TITLE
Consolidate SECURITY_CONTACTS into OWNERS

### PR DESCRIPTION
**Pull request by sfowl**
**Which issue(s) this PR fixes**:

Fixes https://github.com/kubernetes/committee-security-response/issues/149

### CODE REVIEW

File: committee-steering/governance/sig-governance.md [hunk 1]
```diff
@@ -160,8 +160,7 @@ within 30 days from the date of their appointment.
   - *MUST* be a contact point for the Product Security Committee to reach out to
    for triaging and handling of incoming issues
   - *MUST* accept the [Embargo Policy]
-  - Defined in `SECURITY_CONTACTS` files, this is only relevant to the root file
-   in the repository. Template [SECURITY_CONTACTS]
+  - Defined in `security_contacts` field of `OWNERS` files
```
Comment (tallclair) [inline committee-steering/governance/sig-governance.md:163]: Should we add something about having a public email address here? Technically optional, but if we don't have any way of reaching a security contact they're not a very good contact... (crosses over into https://github.com/kubernetes/committee-security-response/issues/56, but a simple documentation line for now might be an improvement)

Comment (tallclair) [inline committee-steering/governance/sig-governance.md:163]: Should we add a note that if no `security_contacts` are specified, we default to the top-level project approvers?

**Review by tallclair — COMMENTED** [review]
Thanks! I'm not sure what the merge order should be, but you should also update the template repo: https://github.com/kubernetes/kubernetes-template-project/blob/main/SECURITY_CONTACTS

**Comment by sfowl** [comment]
> Thanks! I'm not sure what the merge order should be, but you should also update the template repo: https://github.com/kubernetes/kubernetes-template-project/blob/main/SECURITY_CONTACTS

https://github.com/kubernetes/kubernetes-template-project/pull/38

My thinking was this PR into k/community was a good way to track that the overall change is agreed upon, and the all other PRs would merge afterwards.

File: committee-steering/governance/sig-governance.md (hunk 1 above)
Comment (sfowl) [reply committee-steering/governance/sig-governance.md:164]: good idea! done

Comment (sfowl) [reply committee-steering/governance/sig-governance.md:164]: Added

**Comment by PushkarJ** [comment]
Hey @sfowl Thanks for working on this and sharing this on sig-security-tooling slack channel. 

It seems like this update is being made to allow better ways to communicate with maintainers privately for security issues if I read the original issue correctly https://github.com/kubernetes/committee-security-response/issues/56

If `security_contacts` field is going have just Github Ids how does it enable private communication channel with the maintainers / OWNERS of the code ? Is there a planned phase 2 where we will add more details that enables it?

**Comment by sfowl** [comment]
Hey @PushkarJ , yes, there would need to be a second phase to address the private communication aspect. This first phase does have benefits of it's own though, @tallclair gave a good little overview in this comment:

https://github.com/kubernetes/test-infra/pull/25347#issuecomment-1048145454

**Comment by PushkarJ** [comment]
@sfowl sorry have not gotten to this in a bit. Would you be open to bringing this up for discussion in the upcoming SIG Security [Meeting](https://github.com/kubernetes/community/tree/master/sig-security#meetings)? I think we might benefit from feedback from many other folks who might be interested in this

**Pull request closed without merging**